Gem/Code/Source/AutoGen/NetworkPlayerMovementComponent.AutoComponent.xml

```diff
@@ -16,7 +16,8 @@
 
     <Include File="Source/MultiplayerSampleTypes.h"/>
 
-    <NetworkProperty Type="AZ::Vector3" Name="Velocity" Init="AZ::Vector3::CreateZero()" ReplicateFrom="Authority" ReplicateTo="Client" IsRewindable="true" IsPredictable="true" IsPublic="true" Container="Object" ExposeToEditor="false" ExposeToScript="true" GenerateEventBindings="true" Description="Current velocity of this player"/>
+    <NetworkProperty Type="AZ::Vector3" Name="BaseVelocity" Init="AZ::Vector3::CreateZero()" ReplicateFrom="Authority" ReplicateTo="Client" IsRewindable="true" IsPredictable="true" IsPublic="true" Container="Object" ExposeToEditor="false" ExposeToScript="true" GenerateEventBindings="true" Description="Velocity of applied from other sources"/>
```

Comment:
Perhaps call it, VelocityFromExternalSources?

Reply:
We should update the description..